.Rbuildignore

```diff
@@ -17,12 +22,13 @@ demo/.*\.css
 
 # ignore python files/folders
 setup.py
-index.py
-config.py
+usage.py
+setup.py
```

Comment:
Looks like a regression... I remember reviewing this before, but I can't break the commits down because it looks like they were rebased into 28c375bea4306750.

Reply:
Thanks for noting this. Fixed in d7bd0d46f052ffe8a50697947ef7bddc7194854f.